tools/jinja2-filters.py

```diff
@@ -0,0 +1,19 @@
+import base64
```

Comment:
Please change it to:
```python
from base64 import b64decode
```
Because you are using just this one specific method from this library.

Reply:
Done.